app/routes/_auth+/forgot-password/index.tsx

```diff
@@ -90,7 +89,7 @@ export async function action({ request }: DataFunctionArgs) {
 	if (response.status === 'success') {
 		return redirect(redirectTo.toString())
 	} else {
-		submission.error[''] = response.error.message
+		submission.error[''] = [response.error.message]
```

Comment:
The `report` helper is not available yet. But the error has to always be an array now.

Reply:
Sounds good 👍